src/qt/bantablemodel.h

```diff
@@ -45,7 +44,7 @@ class BanTableModel : public QAbstractTableModel
     Q_OBJECT
 
 public:
-    explicit BanTableModel(interfaces::Node& node, ClientModel *parent = nullptr);
+    explicit BanTableModel(interfaces::Node& node, QObject* parent = nullptr);
```

Comment:
nit: not allowing default parent seems safer w.r.t. Qt parent-child properties.
```suggestion
    explicit BanTableModel(interfaces::Node& node, QObject* parent);
```

Reply:
Done, and pushed to #18036